htdocs/api/class/api_objectlinks.class.php

```diff
@@ -0,0 +1,473 @@
+<?php
+/*
+/* Copyright (C) 2025  Jon Bendtsen         <jon.bendtsen.github@jonb.dk>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+use Luracast\Restler\RestException;
+require_once DOL_DOCUMENT_ROOT.'/api/class/api.class.php';
+require_once DOL_DOCUMENT_ROOT.'/core/lib/functions.lib.php';
+require_once DOL_DOCUMENT_ROOT.'/core/class/objectlink.class.php';
+
+
+/**
+ * API that gives shows links between objects in an Dolibarr instance.
+ *
+ * @access protected
+ * @class  DolibarrApiAccess {@requires user,external}
+ */
+class ObjectLinks extends DolibarrApi
+{
+	/**
+	 * @var array   $FIELDS     Mandatory fields, checked when create and update object
```

Comment:
Using on "older" template I think - the variable name is not required in the property documentation.
Also, specify fields as `string[]` for static analysis

Reply:
@mdeweerd "older" ... I originally used v21 as my development base

Reply:
v21 is "older" than develop .... 😄, 2546 commits behind develop.
I can't find a single `@var array   $FIELDS` in develop .